Modules/HMPID/src/HmpidTask.cxx

```diff
@@ -245,7 +246,7 @@ void HmpidTask::monitorData(o2::framework::ProcessingContext& ctx)
       }
       mDecoder->setUpStream(ptrToPayload, (long int)payloadSize);
       if (!mDecoder->decodeBufferFast()) {
-        ILOG(Error, Devel) << "Error decoding the Superpage !" << ENDM;
+        ILOG_INST.log(msgLimit, "Error decoding the Superpage !");
```

Comment:
is there a way of knowing from the logs, that the future messages of this kind are suppressed?

Reply:
When the limit is reached it says: `Warning - [auto-mute] Verbosity threshold exceeded (1 msg < 600 s), discarding next similar messages`

Reply:
cool, thanks!